wolfcrypt/test/test.c

```diff
@@ -21166,8 +21166,6 @@ static int ecc_test_curve_size(WC_RNG* rng, int keySize, int testVerifyCount,
 #endif
 
     ret = wc_ecc_make_key_ex(rng, keySize, userA, curve_id);
-    if (ret != 0)
```

Comment:
Perhaps moving this to after the async wait? Not sure removing these lines is the right fix. Same for 21193 below.

Reply:
it looked like there was duplicate checks on ret, one before wc_AsyncWait and one after. The GUI here is jumping over the second check on ret, line 21194 and 21172 in wolfcrypt/test/test.c

Reply:
Okay my bad, I see it now. GitHub review is not showing the lines with duplicate check below. Good catch.